include/dem/update_local_particle_containers.h

```diff
@@ -63,7 +63,8 @@ void
 update_contact_container_iterators(
   pairs_structure &pairs_in_contact,
   typename DEM::dem_data_structures<dim>::particle_index_iterator_map
-    &particle_container);
+    &        particle_container,
+  const bool load_balance_step = false);
```

Comment:
Is load balance_step the right variable name here or should it be "clear_contact_structures" In essence, the fact that we clear when we load balance is one thing, but the action being asked of the structure is to clear it.